config.php

```diff
@@ -1,5 +1,23 @@
 <?php
-	if ( ! defined( 'DB_HOST' ) ) define( 'DB_HOST', 'hospital.test' );
-	if ( ! defined( 'DB_USER' ) ) define( 'DB_USER', 'root' );
-	if ( ! defined( 'DB_PASSWORD' ) ) define( 'DB_PASSWORD', '' );
-	if ( ! defined( 'DB_NAME' ) ) define( 'DB_NAME', 'hospital' );
\ No newline at end of file
+
+	$whitelist = array(
+		'127.0.0.1',
+		'::1'
+	);
+
+	if ( ! in_array( $_SERVER['REMOTE_ADDR'] , $whitelist ) ) {
+		if ( ! defined( 'DB_HOST' ) ) define( 'DB_HOST', 'localhost' );
+		if ( ! defined( 'DB_USER' ) ) define( 'DB_USER', 'bappidgr_hospita' );
+		if ( ! defined( 'DB_PASSWORD' ) ) define( 'DB_PASSWORD', '0,Wc1CxSgplR' );
+		if ( ! defined( 'DB_NAME' ) ) define( 'DB_NAME', 'bappidgr_hospital' );
+		if ( ! defined( 'BASE_URL' ) ) define('BASE_URL', 'bappi-d-great.com/hospital');
+	    if ( ! defined( 'TABLE_PREFIX' ) ) define('TABLE_PREFIX', 'hp_');
+	}
+	else {
+		if ( ! defined( 'DB_HOST' ) ) define( 'DB_HOST', 'localhost' );
+		if ( ! defined( 'DB_USER' ) ) define( 'DB_USER', 'root' );
+		if ( ! defined( 'DB_PASSWORD' ) ) define( 'DB_PASSWORD', '' );
+		if ( ! defined( 'DB_NAME' ) ) define( 'DB_NAME', 'hospital' );
+		if ( ! defined( 'BASE_URL' ) ) define('BASE_URL', 'hospital.test');
+		if ( ! defined( 'TABLE_PREFIX' ) ) define('TABLE_PREFIX', 'hp_');
```

Comment:
`if ( ! defined( 'TABLE_PREFIX' ) ) define('TABLE_PREFIX', 'hp_');`

This line is same on both conditions, don't need to write twice.